pkg/apis/networking/v1beta1/well_known_annotations.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+const (
+	// AnnotationIngressClass is used to indicate which Ingress controller(s)
+	// should implement this Ingress.
+	AnnotationIngressClass = "kubernetes.io/ingress.class"
```

Comment:
I don't recall ever adding a well_know_annotation before. Does something trip without this or is it syntactic-sugar?

Reply:
Haha yeah, I don't think there's anything special about it. I was just copying the pattern from elsewhere where commonly used labels are dropped into a `well_known_labels.go` file. I think this is a first for annotations though, just wasn't sure where else to put this.